src/Corax/IndexSearcher.cs

```diff
@@ -581,6 +636,189 @@ public MultiTermMatch ContainsQuery<TScoreFunction>(string field, string contain
                     this, new ContainsTermProvider(this, _transaction.Allocator, terms, field, containsTerm), scoreFunction));
         }
 
+        public IQueryMatch SearchQuery(string field, string searchTerm, SearchOperator @operator, int analyzerId)
+        {
+             ReadOnlySpan<byte> term = Encoding.UTF8.GetBytes(searchTerm).AsSpan();
+            
+            var analyzer = _analyzers[analyzerId] ?? throw new InvalidDataException($"{nameof(SearchQuery)} requires analyzer.");
```

Comment:
You need `TryGetValue` here, this will throw a `KeyNotFoundException`

Reply:
Good point, I will change that.